### PR TITLE
Fix for initialization issue in pq_read_arg,orc_read_arg

### PR DESCRIPTION
**Pull request by pradghos**
Fixing - https://github.com/rapidsai/cudf/issues/1878

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by kkraus14** [comment]
ok to test

**Comment by mjsamoht** [comment]
Looks reasonable to me.  

@j-ieong can you take a quick look since you're more familiar?

**Comment by pradghos** [comment]
Thanks for reviewing this PR ; Please let me know in case of any further review comments.

File: python/cudf/bindings/orc.pyx [hunk 1]
```diff
@@ -57,14 +57,15 @@ cpdef cpp_read_orc(path, columns=None, stripe=None, skip_rows=None,
...
+    orc_reader.use_cols_len = 0
     cdef vector[const char*] vector_use_cols
     if usecols is not None:
-        arr_cols = []
         for col in usecols:
             arr_cols.append(str(col).encode())
-        vector_use_cols = arr_cols
-        orc_reader.use_cols = vector_use_cols.data()
         orc_reader.use_cols_len = len(usecols)
+    vector_use_cols = arr_cols
```
Comment (j-ieong) [inline python/cudf/bindings/orc.pyx]: Using an if-else block with the other path be `orc_reader.use_cols = NULL` and `orc_reader.use_cols_len = 0` makes more sense and matches the other sections. But this is fine.

**Pull request merged**